api/admin/controller/metadata_services.py

```diff
@@ -12,9 +12,9 @@
 from core.util.http import HTTP
 from core.util.problem_detail import ProblemDetail
 
-from . import SettingsController
+from . import SitewideRegistrationController
 
-class MetadataServicesController(SettingsController):
+class MetadataServicesController(SitewideRegistrationController):
```

Comment:
This is the core problem -- register_with_metadata_wrangler is in SitewideRegistrationController, not in SettingsController.